parm/ufs/land/noahmptable.tbl

```diff
@@ -0,0 +1,780 @@
+&noahmp_usgs_veg_categories
```

Comment:
Please remove this file from the global-workflow repository and use the one from the ufs-weather-model to link into the `global-workflow/parm/ufs/`. There is no need to create a `land` sub-directory under `parm/ufs`.